docs/programming-oak.md

```diff
@@ -555,14 +555,14 @@ function:
 [embedmd]:# (../examples/abitest/module_0/rust/src/lib.rs Rust /^#.*no_mangle.*/ /pub fn main\(.*/)
 ```Rust
 #[no_mangle]
-pub extern "C" fn frontend_oak_main(handle: u64) {
+pub extern "C" fn frontend_oak_main(in_handle: u64) {
     let _ = std::panic::catch_unwind(|| {
         oak::set_panic_hook();
-        main(handle)
+        main(in_handle);
     });
 }
 
```

Comment:
I think there's a [whole section](https://github.com/project-oak/oak/blob/master/docs/programming-oak.md#per-node-boilerplate) further up that needs updating to match this change.

Reply:
Done, PTAL.